packages/esl/src/esl-popup/core/esl-popup.ts

```diff
@@ -248,7 +242,7 @@ export class ESLPopup extends ESLToggleable {
 
     this.$$off({group: 'observer'});
 
-    memoize.clear(this, ['offsetArrowRatio', '$container']);
+    memoize.clear(this, ['offsetTetherRatio', '$container']);
```

Comment:
`offsetTetherRatio` seems to be not memoized

Reply:
But should be. Fixed. Thanks